modules/web/src/app/wizard/template.html

```diff
@@ -13,11 +13,20 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<form [formGroup]="form">
+
+<ng-container *ngIf="loadingClusterTemplate">
+  <mat-spinner [diameter]="50"
+               class="km-spinner"
+               color="accent"></mat-spinner>
+</ng-container>
+
+<form [formGroup]="form"
+      *ngIf="!loadingClusterTemplate">
   <mat-card fxFlex="100%">
```

Comment:
Small thing but we need to show `Customize Template` header here (as mentioned in https://github.com/kubermatic/dashboard/issues/4948) when editing cluster template.